### PR TITLE
Special quests

### PR DESCRIPTION
**Pull request by Gorzen**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java]: This is an old file, should be deleted

File: app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java [hunk 2]
```diff
@@ -204,6 +209,10 @@ public void addQuestion(final Question question) {
         db.collection(FB_QUESTIONS).document(question.getQuestionId()).set(questionData);
     }
 
+    public void updateQuests(Map<String, Object> remotePlayerData) {
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java]: Is it an oversight? Is it normal?

File: app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```
Comment (JulesssG) [inline app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java]: Why not put all adapter (list and view in utils) in there?

Comment (nheyek) [reply app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java]: I think this indeed should be done.

File: app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java (hunk 2 above)
Comment (nheyek) [reply app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java]: I did not put this in here.

Comment (nheyek) [reply app/src/main/java/ch/epfl/sweng/studyup/firebase/Firestore.java]: Strange. I did not write this, not sure how it ended up here.

File: app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java (hunk 1 above)
Comment (nheyek) [reply app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java]: This is a package containing the list view for Questions and Special Quests.

**Pull request merged**